docs/guide/samples-howto.md

```diff
@@ -1,7 +1,7 @@
 ## How To
 
 This section gives various examples of complex behaviors and/or functionalities.
-The world files are located in the "[WEBOTS\_HOME/projects/samples/howto/worlds]({{ url.github_tree }}/projects/samples/howto/worlds/)" directory, and their controllers in the "[WEBOTS\_HOME/projects/samples/howto/controllers]({{ url.github_tree }}/projects/samples/howto/controllers/)" directory.
+The examples are located in "[WEBOTS\_HOME/projects/samples/howto/]({{ url.github_tree }}/projects/samples/howto/)". Each example has his own controllers and worlds directory.
```

Comment:
There is a typo here: "his" &rarr; "its". Also, "directory" &rarr; "directories".